pkg/objectupdate/rte/rte.go

```diff
@@ -109,6 +110,20 @@ func DaemonSetHashAnnotation(ds *appsv1.DaemonSet, cmHash string) {
 	template.Annotations[hash.ConfigMapAnnotation] = cmHash
 }
 
+func DaemonSetArgs(ds *appsv1.DaemonSet) error {
+	cnt, err := FindContainerByName(&ds.Spec.Template.Spec, MainContainerName)
+	if err != nil {
+		return err
+	}
+	flags := flagcodec.ParseArgvKeyValue(cnt.Args)
+	if flags == nil {
+		return fmt.Errorf("cannot modify the arguments for container %s", cnt.Name)
+	}
+	flags.SetToggle("--pods-fingerprint")
```

Comment:
This would mean that if we have the `pods-fingerprint` flag enabled in the RTE daemonset, it would be disabled. Seems a bit odd to me. Why don't we add an explicit flag for the podfingerprint in the NROP spec that gets passed to RTE?

Also the other way round, if I am a consumer of this with a not very dynamic cluster and want to disable this capability. Does this mean that the way I have to do that is go [here](https://github.com/openshift-kni/numaresources-operator/blob/92957803de8e8180824ae0761ba870850144bb6b/rte/main.go#L123-L124) and change the flags `true`?

Reply:
I didn't though about possibly disabling the fingerprint, which is anyway a good point. I wonder what could be a possible way to expose this option, which I expect to be left as default (thus enabled) >= 90% of the times

Reply:
ack, we can add a per-nodegroup disable flag for `pods-fingerprint` and tune the per-nodegroup daemonsets accordingly. I'm not sure we need to add `expose-timing` however. U/S added ( :) ) to make the e2e tests more robust and easier to write, I don't see much value in enabling this flag in our case.

Reply:
Also, I noticed that `expose-timing` flag here was not toggled. Was that intentional?

Reply:
yes, `expose-timing` is supported but not enabled by default.

Reply:
Thanks, ignore my last comment. You already answered the question. I think github didn't refresh and show your latest comment.

Reply:
> we can add a per-nodegroup disable flag for `pods-fingerprint` and tune the per-nodegroup daemonsets accordingly.
I was actually thinking about enabling/disabling this feature for all the nodegroups but tuning per-nodegroup daemonsets is taking it a step further and better than I thought originally :)